locust/test/test_dispatch.py

```diff
@@ -2214,15 +2335,17 @@ class User2(User):
 
         user_classes = [User1, User2]
 
+        target_user_count = TargetUserCount(self.user_dispatcher_class, user_classes)
+
         worker_node1 = WorkerNode("1")
         worker_node2 = WorkerNode("2")
 
         worker_nodes = [worker_node1, worker_node2]
 
-        users_dispatcher = UsersDispatcher(worker_nodes=worker_nodes, user_classes=user_classes)
+        users_dispatcher = self.user_dispatcher_class(worker_nodes=worker_nodes, user_classes=user_classes)
 
         # user count = 1
-        users_dispatcher.new_dispatch(target_user_count=1, spawn_rate=1)
+        users_dispatcher.new_dispatch(target_user_count=target_user_count(1), spawn_rate=1)
```

Comment:
Uh. What? `target_user_count(1)` instead of just `1`? I dont understand it and I'm pretty sure if I did make the effort to understand it I wouldnt like it :)